meson.build

```diff
@@ -83,6 +83,7 @@ dbusmenu_gtk = dependency('dbusmenu-gtk3-0.4', required: get_option('dbusmenu-gt
 giounix = dependency('gio-unix-2.0', required: get_option('dbusmenu-gtk'))
 jsoncpp = dependency('jsoncpp')
 pugixml = dependency('pugixml')
+libinput = dependency('libinput')
```

Comment:
Why? I don't see libinput references in the code.

Reply:
See also 74db69dcb7cc

Reply:
FWIW, this PR builds fine if libinput is removed from meson.build and libinput package deinstalled.